jetty-server/src/main/java/org/eclipse/jetty/server/handler/gzip/GzipHandler.java

```diff
@@ -153,7 +153,7 @@
 {
     public static final String GZIP = "gzip";
     public static final String DEFLATE = "deflate";
-    public static final int DEFAULT_MIN_GZIP_SIZE=16;
+    public static final int DEFAULT_MIN_GZIP_SIZE=2048;
```

Comment:
Seems unrelated.

Reply:
This one is related.  We had two different defaults, one in XML and one in code.  In this case I decided to make the default just in the code and the change was needed to pass some tests.